cura/CuraApplication.py

```diff
@@ -843,6 +845,7 @@ def _loadPlugins(self) -> None:
         self._plugin_registry.addType("profile_reader", self._addProfileReader)
         self._plugin_registry.addType("profile_writer", self._addProfileWriter)
         self._plugin_registry.addType("backend_plugin", self._addBackendPlugin)
+        self._plugin_registry.addType("view_manipulator", self._addViewManipulator)
```

Comment:
I can't seem to find any documentation what a view manipulator is supposed to do. So for future reference, we should describe that somewhere.